deployments/liqo/values.yaml

```diff
@@ -37,6 +37,8 @@ networking:
   # The default value is configured to ensure correct behavior regardless of the combination of the underlying environments
   # (e.g., cloud providers). This guarantees improved compatibility at the cost of possible limited performance drops.
   mtu: 1340
+  # -- Select the mode to enforce security on connectivity among clusters 
+  securityMode: "IntraClusterTrafficSegregation"
```

Comment:
Adds the possible values in the comment. I don't remember what we decided about the default value, is it correct? Because the flag has the full-pod-to-pod value as default.